src/tests/screens/SurveyDetail/QuestionBuilders/determineQuestionType.test.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import 'tests/__mocks__/matchMedia';
+import DetermineQuestionType from 'screens/SurveyDetail/QuestionBuilders/determineQuestionType';
+
+describe('given DetermineQuestionType is mounted', () => {
+  describe("given the question has type 'heart'", () => {
+    it('renders the Rating component', () => {
+      const testId = 'rating-list'
+      const type = 'heart'
+      const pick = 'one'
+
+      const { getByTestId } = render(
+        <DetermineQuestionType type={ type } pick={ pick } />
+      )
+
+      expect(getByTestId(testId)).toBeInTheDocument();
```

Comment:
@rafayet-monon Should we also expect the specific emoji to be rendered?

Reply:
@carryall added in [8c6a30b](https://github.com/rafayet-monon/surveyor/pull/29/commits/8c6a30b3eea3defe47eea260928efa8002013686)